### PR TITLE
Fixup 6612, DOC: typo in the docstring of `random.multinomial`

### PR DESCRIPTION
**Pull request by charris**
Update of #6612.

### CODE REVIEW

File: numpy/random/mtrand/mtrand.pyx [hunk 1]
```diff
@@ -4461,8 +4461,22 @@ cdef class RandomState:
 
         A loaded dice is more likely to land on number 6:
```
Comment (rkern) [inline numpy/random/mtrand/mtrand.pyx]: While we're here: `dice -> die`.

**Comment by charris** [comment]
Done.

**Comment by rkern** [comment]
LGTM!

**Comment by charris** [comment]
Thanks Robert.

**Pull request merged**